### DISCUSSION TITLE
#8511 moved circuit terminations to bottom of page

### DISCUSSION THREAD

**Discussion by PieterL75** [netbox-community/netbox#10612] [General]
https://github.com/netbox-community/netbox/issues/8511 made a change to the circuit page.
termination A and Z are now way at the bottom of the page, and almost fully hidden.

I preferred the layout where the terminations were displayed in the right side.

Is it an option to restore this ?
@ryanmerolle 

Pieter

**Comment by PieterL75** [comment]
Looks like I'm the only one :-) 
Too bad.. it really is hiding the most import info at the bottom..

**Comment by seismiccollision** [comment]
I agree with you. The terminations and contacts are the primary reason I navigate to a circuit page. I'd vote for the layout:
Left column, top to bottom: Circuit, Contacts, Tags, Comments, Custom fields
Right column, top to bottom: Termination - A Side, Termination - Z Side, Images

**Comment by jeremystretch** [comment]
I think my intent was to put them side-by-side for easier comparison, but your point about keeping them "above the fold" is valid. I've opened #10685 to address this.

**Reply by seismiccollision** [reply]
Thanks a bunch!
